### PR TITLE
Feature/tinyswing 48

### PR DESCRIPTION
**Pull request by shikhanansi**
Added new page for Swing integration as requested in https://ephocks.atlassian.net/browse/TINYSWING-48. @tinydylan @jscasca  please review.

### CODE REVIEW

File: integrations/swing.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+layout: default
+title: TinyMCE Swing integration
+title_nav: Swing
+description: Seamlessly integrates TinyMCE into Swing applications.
+keywords: integration integrate swing
+---
+
+Users can easily configure TinyMCE in Swing through the TinyMCE Swing Integration. 
+
+### Buy TinyMCE Swing integration
```
Comment (tinydylan) [inline integrations/swing.md]: We've used the product name "TinyMCE for Swing" in other text.

File: integrations/swing.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+To start using TinyMCE in Swing as your new rich text editor, the first step is to obtain a copy of the Integration. 
+
+Contact [customer support](https://www.tiny.cloud/contact/) to get started with a customized TinyMCE Swing integration for your specific use case.
+
+### Other resources
+
+* [GitHub repository](https://github.com/tinymce/tinymce-swing-codesamples)
+
+* An additional set of documentation is shipped with the integration in `zip` files. These files contain a library of `java` docs and API reference guides which help in understanding and applying the concepts.
```
Comment (tinydylan) [inline integrations/swing.md]: The integration is packaged as a zip file. The zip file contains:
- readme.txt - basic instructions
- license.txt
- release-notes.txt
- the jar files that implement the integration
- JavaDocs - "JavaDoc" or "JavaDocs" is well-known amongst Java developers. `javadoc` is a tool that produces web pages of docs from specially-formatted comments in the code.

File: integrations/swing.md (hunk 1 above)
Comment (shikhanansi) [reply integrations/swing.md]: Fixed

File: integrations/swing.md (hunk 2 above)
Comment (shikhanansi) [reply integrations/swing.md]: Fixed

**Pull request merged**